distribution/lib/Standard/Base/0.0.0-dev/src/System/File.enso

```diff
@@ -416,6 +488,14 @@ type File
     name : Text
     name = this.prim_file.getName
 
+    ## Returns the extension of the file
+    extension : Text
+    extension =
```

Comment:
```suggestion
    ## Returns the extension of the file.
    extension : Text
    extension =
```

Would appreciate an example, because reading the code it is not immediately clear if `foo.txt . extension` gives `txt` or `.txt`.